test/checkpointctl.bats

```diff
@@ -263,16 +262,14 @@ function teardown() {
 	[[ ${lines[10]} == *"proc"* ]]
 }
 
-@test "Run checkpointctl inspect with tar file and --mounts and --full-paths and valid spec.dump" {
+@test "Run checkpointctl inspect with tar file and --ps-tree and missing pstree.img" {
 	cp data/config.dump "$TEST_TMP_DIR1"
 	cp data/spec.dump "$TEST_TMP_DIR1"
 	mkdir "$TEST_TMP_DIR1"/checkpoint
 	( cd "$TEST_TMP_DIR1" && tar cf "$TEST_TMP_DIR2"/test.tar . )
-	checkpointctl inspect "$TEST_TMP_DIR2"/test.tar --mounts --full-paths
```

Comment:
Would it be possible to apply these changes in the previous commit (test: add tests for inspect sub-command)? This would ensure that `make test` will pass after each commit and make it easier to review the pull request.

It would be good to mention why `--full-paths` has been removed in the commit message introducing the `inspect` command (inspect: add new sub-command inspect).